localization_node:
  ros__parameters:
    beacon_radius: 0.5
    beacon_count: 3
    beacon_positions: [
       1.0, 1.5,
      -1.0, 1.5,
       0.0, -1.5]
    beacon_range_correction_factor: 0.75